src/main/java/com/atlassian/jira/cloud/jenkins/util/WebhookUrlValidator.java

```diff
@@ -0,0 +1,17 @@
+package com.atlassian.jira.cloud.jenkins.util;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+public final class WebhookUrlValidator {
+
+    public static boolean isValid(final String webhookUrl) {
+        try {
+            new URL(webhookUrl).toURI();
```

Comment:
I wish there was a nicer way to validate a URL in Java without some external library 🤷🏼